ocaml/xapi-consts/api_messages.ml

```diff
@@ -353,3 +353,31 @@ let failed_login_attempts = addMessage "FAILED_LOGIN_ATTEMPTS" 3L
 
 let tls_verification_emergency_disabled =
   addMessage "TLS_VERIFICATION_EMERGENCY_DISABLED" 3L
+
+exception InvalidMessagePrio of int64
+
+let css_expiring prio =
+  let prio_str =
+    match prio with
+    | 1L ->
+        "CRITICAL"
+    | 2L ->
+        "MAJOR"
```

Comment:
This priority to Message level suffix map is not related to css, 
and thus can be generalized

Reply:
I think it can be generalised once it's needed for more cases. I don't think it;s needed for this PR.

Ideally I'd like a Priority module:

```ocaml
module Priority = struct
  type t = Critical | Major | Warning | Minor | Informational

  let to_string = function
  | Critical ->
      "Critical"
  ...

  let to_int_64 = function
  | Critical ->
      1L
  ...
end

let addMessage name priority =
  let msg = (name Priority.to_int64 priority)
  let _ = msgList := msg :: !msgList in
  msg

let ha_statefile_lost addMessage "HA_STATEFILE_LOST" Priority.Critical

...
```

Reply:
It is not used for more cases,  for now,  but it can be generalize  for future usage, then you do not need worry about  future new mapping again.

The `Priority` module above also expose priority level (Critical, Major) than number 1L, 2L, which looks better.